raphtory-graphql/src/model/graph/edge.rs

```diff
@@ -351,22 +356,22 @@ impl GqlEdge {
         GqlEdges::new(self.ee.explode_layers())
     }
 
-    /// Returns a list of timestamps of when an edge is added or change to an edge is made.
+    /// Returns a History object with time entries for when an edge is added or change to an edge is made.
     ///
     /// Returns:
-    ///     List[int]:
-    async fn history(&self) -> Vec<i64> {
+    ///     History:
+    async fn history(&self) -> GqlHistory {
         let self_clone = self.clone();
-        blocking_compute(move || self_clone.ee.history()).await
+        blocking_compute(move || self_clone.ee.history().into()).await
```

Comment:
As history is now lazy, no need to run this with blocking compute any more? Or are there still scenarios where this triggers non-trivial computation?